### ISSUE TITLE
Slowdown doesn't work in possibly many scenes

### ISSUE THREAD

**Issue by drojf** [drojf/ai_somnium_fps_camera#3]
The slowdown mode completely doesn't work in the final cutscene of the yellow route (top left route), and probably other scenes.

I tried some other methods to set slowdown like constantly forcing the value, and even using the games internal functions but it doesn't seem to affect it. I'm wondering if the Lua scripting portion of the game is somehow responsible for it, as the game must be able to increase speed for the skip function.

I might need to know more unity stuff to fix this, as I did see a "time" property on the animations,  which might ignore Unity's global slowdown

edit: Calling the Pause() of the RootNode does pause the game, but it then freezes completely, even if you try to call UnPause() (or maybe inputs are disabled??). Pause/unpause might be another alternative, though.

edit2: It was the Pause() function disabling the input handler code which caused it to freeze, but even after unpausing the screen remains black.....

edit3: If I delete the part of the Pause() function which pauses the monobehaviors, it seems to work as expected!

**Comment by drojf** [comment]
Ok, it looks like doing the following allows you to completely pause/play the cutscene:

1. Insert the following code somewhere, to execute the Pause() and Unpause() functions

```c#
	if (Input.GetKeyDown(KeyCode.F2))
	{
		RootNode[] array2 = UnityEngine.Object.FindObjectsOfType<RootNode>();
		for (int i = 0; i < array2.Length; i++)
		{
			array2[i].Pause();
		}
	}
	else if (Input.GetKeyDown(KeyCode.F3))
	{
		RootNode[] array3 = UnityEngine.Object.FindObjectsOfType<RootNode>();
		for (int j = 0; j < array3.Length; j++)
		{
			array3[j].UnPause();
		}
	}
```

2. (PLEASE READ DISCLAIMER) In the Pause() function, remove the part which disables the monobehaviors, and disable the part which disables the cameras
- The disclaimer with this part is that it's probably better to make a copy the entire Pause() function, rather than modifying it (something like "ModPause()). This will prevent interfering with the game's existing pause function.

This allows you to completely pause the game even while piloting the camera.

**Comment by drojf** [comment]
Should be fixed as of  214ec60
